style.css

```diff
@@ -0,0 +1,760 @@
+    padding: 50px 100px;
+    background-size: cover;
+    height: auto;
+    width: auto;
+    gap: 10px;
+  }
+
+  .packages-title {
+    font-size: 18px;
+    font-weight: 900;
+    margin: 0;
+    padding: 0 0 25px 0;
+  }
+
+  .card-stack {
+    flex-direction: row;
+    gap: 7px;
+    height: auto;
+    width: 100%;
+  }
+
+  .card-stack .cards {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    width: auto;
+    height: auto;
+    gap: 10px;
+    padding: 10px 10px;
+    border-radius: 0.5rem;
+  }
+
+  .cards-img {
+    padding: 0;
+  }
+
+  .card-title {
+    text-align: center;
+    width: auto;
+    padding: 0;
+  }
+
+  .card-text {
+    width: auto;
+    text-align: center;
+    height: auto;
+    line-height: 22px;
+    padding: 0;
+  }
+
+  .view-all {
+    display: flex;
+    width: auto;
+    height: auto;
+    font-size: 14px;
+    font-weight: 400;
+    color: #fff;
+    padding-top: 25px;
+  }
+
+  #speaker-cards {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap: 5%;
+    width: auto;
+    height: auto;
+    margin: 50px;
+  }
+
+  .each-card {
+    display: flex;
+    gap: 20%;
+    width: 100%;
+    height: auto;
+    padding: 0;
+  }
+
+  .speaker-images {
+    width: 120px;
+    height: 120px;
+  }
+
+  .speaker-bgimages {
+    width: 60px;
+    height: 60px;
+    border-radius: 50%;
+    margin-left: -50%;
+  }
+
+  .card-body {
+    text-align: left;
+    width: 100%;
+  }
+
+  .speaker-title {
+    color: #ec5242;
+    width: 100%;
+  }
+
+  .speaker-bio {
+    width: 100%;
+  }
+
+  .view-more {
+    display: none;
+  }
+
+  .join-us {
+    display: none;
+  }
+
+  #footer1 {
+    display: flex;
+    flex-direction: column;
+    background-color: #292929;
+    height: auto;
+    padding: 0;
+  }
+
+  .sponsors-title {
+    text-align: center;
+    font-family: 'Lato', sans-serif;
+    font-weight: 700;
+    font-size: 18px;
+    padding: 3%;
+    color: #d3d3d3;
+    text-decoration-line: underline;
+    text-decoration-color: #ec5242;
+    text-underline-offset: 10px;
+  }
+
+  .sponsorship {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 10% 5% 10%;
+    font-family: 'Lato', sans-serif;
+    font-weight: 700;
+    font-size: 32px;
+    color: #d3d3d3;
+  }
+
+  .sponsors-companies {
+    font-weight: 400;
+    font-size: 8px;
+    color: #d3d3d3;
+    list-style: none;
+  }
+
+  #footer2 {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 10%;
+    padding: 5% 30%;
+  }
+
+  .footer2-logo {
+    width: 64px;
+    height: 64px;
+    padding-left: 0;
+  }
+
+  .footer2-p {
+    font-family: 'Lato', sans-serif;
+    font-size: 14px;
+    font-weight: 500;
+    font-style: normal;
+    line-height: 20px;
+    text-align: left;
+    width: 80%;
+  }
+}
+
+@media screen and (min-width: 1024px) {
+  .main-logo {
+    margin-right: 350px;
+  }
```

Comment:
- [ ] **As mentioned by the previous reviewer,** there is an issue **with your logo disappearing when transitioning from the mobile to the desktop version**. The problem might be related to the fixed `margin-right` that you applied to it. I recommend **checking this and considering an alternative approach**. One possible solution could be to create a separate **div** for the other elements and apply a `display flex` to the entire nav. This can help ensure consistent visibility and alignment across different screen sizes. 😉

  _Please watch the following video for a better understanding_ 🎬
  
  [5_24_2023_4-06-14 PM.webm](https://github.com/MariamOsman11/CAPSTONE-PROJECT-ONE/assets/107952724/7dd6a3c6-8bea-4093-9007-1c7483341978)